torch/nn/utils/stateless.py

```diff
@@ -128,12 +196,31 @@ def functional_call(
             >>> print(mod.foo)  # tensor(0.)
             >>> print(a['foo'])  # tensor(1.)
 
+    .. note:: If the module has tied weights, whether or not the reparameterization respects the tying is determined
```

Comment:
nit: I don't think there is any concept of "reparameterization" from the user point of view here.

Reply:
Changed to functional_call